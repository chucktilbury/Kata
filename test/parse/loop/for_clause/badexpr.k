;-------------------------------------------------------------------------------
; Demonstrate bad expression error
;-------------------------------------------------------------------------------

func name ()() {
    for ( name in dict something != 0 ) {

    }
}
