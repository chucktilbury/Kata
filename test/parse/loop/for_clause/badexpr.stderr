Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badexpr.k: 6: 23: expected an expression but got dict
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badexpr.k: 6: 23: expected function body element but got for
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badexpr.k: 6: 23: expected function body but got for
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badexpr.k: 6: 23: expected module item or end of file but got func
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badexpr.k: 6: 23: module cannot be empty
